tools/phenomenal/nmr/bruker2batman.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--Proposed Tool Section: [File Handling]-->
+<tool id="bruker2batman" name="bruker2batman Converter">
```

Comment:
Can we please add a version here? Please see other tool wrappers in the same nmr folder.